awx/ui/src/screens/WorkflowApproval/WorkflowApprovalDetail/WorkflowApprovalDetail.js

```diff
@@ -194,6 +258,88 @@ function WorkflowApprovalDetail({ workflowApproval }) {
           value={secondsToHHMMSS(workflowApproval.elapsed)}
         />
       </DetailList>
+      <Title headingLevel="h2">{t`Workflow job details`}</Title>
+      <Divider />
+      <WFDetailList gutter="sm">
+        <Detail
+          label={t`Workflow Job`}
+          value={
+            sourceWorkflowJob && sourceWorkflowJob?.id ? (
+              <Link to={`/jobs/workflow/${sourceWorkflowJob?.id}`}>
+                {`${sourceWorkflowJob?.id} - ${sourceWorkflowJob?.name}`}
+              </Link>
+            ) : (
+              t`Deleted`
+            )
+          }
+          dataCy="wa-detail-source-job"
+        />
+        {workflowJob?.limit ? (
+          <Detail
+            label={t`Limit`}
+            value={workflowJob.limit}
+            dataCy="wa-detail-source-job-limit"
+          />
+        ) : null}
+        {workflowJob?.scm_branch ? (
+          <Detail
+            label={t`Source Control Branch`}
+            value={workflowJob.scm_branch}
+            dataCy="wa-detail-source-job-scm"
+          />
+        ) : null}
+        {workflowJob?.summary_fields?.inventory ? (
+          <Detail
+            label={t`Inventory`}
+            value={
+              workflowJob.summary_fields?.inventory ? (
```

Comment:
I don't think we need this additional check for `workflowJob.summary_fields?.inventory` since we check for the value in the ternary above.